test/integration/test_http.py

```diff
@@ -73,5 +73,196 @@ def test_basicTxn(self):
         print(result)
         self.assertEqual(result["status"]["code"], 200);
 
+    # Helper method to populate and send request with default values
+    def sendReq(self, api, txn, partitionKeyName="partitionKey", partition="ptest2",
+                rangeKeyName="rangeKey", range_key="rtest2",
+                dataFieldName="data", data="mydata2",
+                coll="HTTPClient", schema="test_schema", ver=1):
+        if api ==  "/api/BeginTxn":
+            request = {}
+        elif api == "/api/Write":
+            record = {partitionKeyName: partition, rangeKeyName: range_key,
+                      dataFieldName: data}
+            request = {"collectionName": coll, "schemaName": schema, "txnID": txn, "schemaVersion": ver, "record": record}
+        elif api == "/api/Read":
+            record = {partitionKeyName: partition, rangeKeyName: range_key}
+            request = {"collectionName": coll, "schemaName": schema, "txnID": txn, "schemaVersion": ver, "record": record}
+        elif api == "/api/EndTxn":
+            # Commit
+            request = {"txnID": txn, "commit": True}
+            print(request)
+        else:
+            raise ValueError('Invalid api' + api)
+
+        url = args.http + api
+        r = requests.post(url, data=json.dumps(request))
+        result = r.json()
+        print(result)
+        return result
+
+    def test_schema_validation(self):
```

Comment:
not sure why this is a test called "schema_validation". It is doing transaction ops, not validating schema

Reply:
Changed to `test_validation`